pkg/kritis/crd/securitypolicy/violation.go

```diff
@@ -22,48 +22,54 @@ import (
 	"github.com/grafeas/kritis/pkg/kritis/apis/kritis/v1beta1"
 	"github.com/grafeas/kritis/pkg/kritis/constants"
 	"github.com/grafeas/kritis/pkg/kritis/metadata"
-)
-
-type Reason string
-
-// A list of security policy violations
-// TODO: Add Attestation checking violations
-const (
-	UnqualifiedImageViolation int = iota
-	FixUnavailableViolation
-	SeverityViolation
+	"github.com/grafeas/kritis/pkg/kritis/policy"
 )
 
 // Violation represents a vulnerability that violates an ISP
 type Violation struct {
 	Vulnerability metadata.Vulnerability
-	Violation     int
-	Reason        Reason
+	VType         policy.ViolationType
```

Comment:
Do you mind finding less of a stuttering name than violation.VType, such as violation.Type?

Reply:
Type is a keyword and i could not use it. hence i named is vType.

Reply:
i can double check this. because last i tried, i was using `type` vs 'Type`

Reply:
yes! `Type` works but `type` does not. Since i changed these to be a private variables i renamed it as vType.